frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -572,9 +578,9 @@ public String dumpConfigurations() {
                 + "\nMetrics dir: "
                 + getCanonicalPath(System.getProperty("METRICS_LOCATION"))
                 + "\nNetty threads: "
-                + getNettyThreads()
+                + (getNettyThreads() == 0 ? DEFAULT_EVENT_LOOP_THREADS : getNettyThreads())
```

Comment:
Can't we simply return the default value set in Netty's MultithreadEventLoopGroup class?

Reply:
@harshbafna I did not want to change the way Netty calculate the value. I could have returned `2*runtime.availableProcessors()`.  For example, in future if they change the value returned the function `NettyRuntime.availableProcessors`, then the value printed by us will not be consistent with they function they use to calculate default.